src/translations/screens/subscreens/Login.ts

```diff
@@ -30,6 +45,18 @@ const LoginTexts = {
       '...to store payment cards for later use. Soon, login will store personal preferences to your profile so that you can carry them with you when swapping phones.',
     ),
   },
+  vipps: {
+    errors: {
+      outdated_app_version: _(
+        'Er du sikker på at telefonnummeret er korrekt?',
+        'Are you certain the phone number is correct?',
+      ),
```

Comment:
The `outdated_app_version` error gives "Are you certain the phone number is correct"?

Reply:
Oos..copy paste mistake 😅  Would fix this.